plugins/generator-1.21.1/neoforge-1.21.1/templates/fluid/fluidbucket.java.ftl

```diff
@@ -39,7 +39,9 @@ public class ${name}Item extends BucketItem {
 
 	public ${name}Item() {
 		super(${JavaModName}Fluids.${REGISTRYNAME}.get(),
-			new Item.Properties().craftRemainder(Items.BUCKET).stacksTo(1).rarity(Rarity.${data.rarity}));
+			new Item.Properties().craftRemainder(Items.BUCKET).stacksTo(1)
+			<#if data.rarity != "COMMON">.rarity(Rarity.${data.rarity})</#if>
+			);
```

Comment:
In the rest of templates, we usually align ) closure with opening statement, meaning one tab less in this case.

Example:

![image](https://github.com/user-attachments/assets/679cee27-5ba9-471e-be98-c36b7303c020)